backend/config/keys.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by pancho111203 on 15/02/16.
+ */
+
+var keys = {};
+
+keys.clientId = "5546933a86952d67f3c0";
+keys.clientSecret = "71a524daac2a72d20a94a28ae13032f77ea19170";
```

Comment:
We should hide the final keys from github(this ones are temporary). I don't know what is the best way, maybe environment variables?